src/utils/minFee.ts

```diff
@@ -2,7 +2,6 @@ import { BigNumber } from 'bignumber.js'
 import { logDebug } from './miscellaneous'
 
 export const DEFAULT_GAS_PRICE = 40e9 // 40 Gwei
-export const SUBSIDIZE_FACTOR = 1 // no subsidy
 
 // account for a 20% change in the time it takes to mine the tx, and start the batch
 export const BUFFER_MULTIPLIER = 1.2
```

Comment:
well, I guess after this PR is not too relevant since you don't use it anymore. Agree with this PR that is good to keep the calculation around just in case come!